docs/querying/sql.md

```diff
@@ -423,3 +423,12 @@ SELECT count(city) from druid.table where SCALAR_IN_ARRAY(city, ?)
 ```
 
 sample java code using dynamic parameters is provided [here](../api-reference/sql-jdbc.md#dynamic-parameters).
+
+## Reserved keywords
+
+There are reserved keywords in Apache Druid. To use reserved keywords in queries, enclose them in quote marks.
```

Comment:
Maybe refer to "Druid SQL" instead of "Apache Druid" here. Can we turn this into an active sentence also? Druid SQL reserves certain keywords for...

Reply:
Had a go at rephrasing this line, please let me know what you think @vtlim. Thanks again for the feedback